base.Dockerfile

```diff
@@ -1,6 +1,7 @@
 # Dockerfile to bootstrap build and test in openshift-ci
 
-FROM openshift/origin-release:golang-1.15
+# TODO: do we need to use quay due to dockerhub rate limiting?
+FROM openshift/origin-release:golang-1.16
```

Comment:
A quick note on this:

- No changes made to the build root Dockerfile will propagate to tests until it's been merged to master first, so we won't get any CI feedback from this, which should be fine but something to call out.
- We may want to avoid using an image hosted on docker.io due to potential flakes from rate limitations.